docs/topics/basic-types.md

```diff
@@ -316,6 +312,11 @@ Unsigned types are implemented using feature that's not yet stable, namely [inli
 
 #### Unsigned arrays and ranges 
 
+> Unsigned arrays and operations on them are in [Beta](components-stability.md). They are almost stable, but migration
```

Comment:
This seems to give quite more strict guaranties than it was before:
> It's up to you to decide if your clients have to explicitly opt-in into usage of your API, but keep in mind that unsigned types are not a stable feature, so API which uses them can be suddenly broken by changes in the language.

Reply:
We now have standard warning messages for Experimental, Alpha, and Beta features based on descriptions in [Stability of Kotlin components](https://kotlinlang.org/docs/components-stability.html). 
However, I remember that you said unsigned arrays are actually experimental and we just called them Beta before the levels were strictly defined. 
So, if the previous wording reflects the current state accurately, we can use it as a custom warning for this unsigned arrays. Should we?